### DISCUSSION TITLE
Is There a Way to Make a Custom Type on Migration

### DISCUSSION THREAD

**Discussion by niitou** [SeaQL/sea-orm#1610] [Q&A] [answered]
Is there a way to create custom type like Postgres composite type with Sea ORM migration ?

![image](https://user-images.githubusercontent.com/57691998/233930398-e3cda899-738c-46f6-9e2c-0f865f0de6f6.png)
I want to make like something above but can't find how to do it in the documentation.

**Comment by billy1624** [comment, marked as answer]
Hey @StevenBudi, absolutely, you can create a column of custom type, https://docs.rs/sea-query/latest/sea_query/table/enum.ColumnType.html#method.custom
